src/ol_concourse/pipelines/libraries/open_api_clients.py

```diff
@@ -66,9 +69,9 @@ def _read_script(script_name: str) -> str:
 )
 mit_open_api_clients_repository = ssh_git_repo(
     name=Identifier("mit-open-api-clients"),
-    uri="git@github.com/mitodl/open-api-clients.git",
+    uri="git@github.com:mitodl/open-api-clients.git",
     branch="main",
-    private_key="((git-private-key))",
+    private_key="((open_api_clients.odlbot_private_ssh_key))",
```

Comment:
We should probably make this more general since we can make use of that key outside of just this pipeline.

Reply:
Agreed. Where should we move it?

Reply:
Gonna say this is a problem for another day for now.